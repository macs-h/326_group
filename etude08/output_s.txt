11000010111011010100000000000000
01000010111011010100000000000000
