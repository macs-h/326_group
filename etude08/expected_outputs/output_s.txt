1-10000101-11011010100000000000000
0-10000101-11011010100000000000000
0-10001000-01000110100101011000000
0-10001011-11110100100011100000000
0-10001000-00101101100100000000000
<<<<<<< HEAD
=======
1
2
3
4
>>>>>>> 53ed6caf0e7efc8afa624a91af11de2e2271ad4b
